### ISSUE TITLE
Hi!! not issue here, just dont know how to contact you

### ISSUE THREAD

**Issue by popoyeLetal** [linuxer9/sc-ng#1]
Im concerned about vac and osiris, I use VAC bypass, own injector +osiris (github version)
I saw you comented dll was being detected, and using a personal build is a solution. My question is how much has to be modified the original dll not to be detected?? Can i just add a few lines, compile it and introduce it into memject... Or it has to bee a long modification in order to make the dll undetectable. 
Thanks for your time!!

**Comment by linuxer9** [comment]
last year, I used a modified version of osiris, and modified version of randoinjector, and played with it on multiple non prime, account it worked great for about 3 months, no vac bans
until one of my accounts got OW ban for obvious cheating, then all other accounts got vac banned.
If I understand VAC correctly, it somehow detects you injected something, even if you use undetected dll, but still can't ban you for just that, as the dll could be unharmful, so it scans signature of that dll and flags your account, If you got OW ban, then it says ahah that dll is for sure a cheating software.
the same behaviour occured before with esportal anti cheat, because this anti cheat injects it self to CS GO the same way as cheats, then may be some one got OW ban and coincendently used esportal, this was enough for vac to initiate vac wave on all esportal users.
you may wonder, why the hell this doesn't happen to public paid cheat? of course many of these P2C users are getting OW bans
because of the use of **VAC bypass** , nearly all P2C I know uses some sort of VAC bypass, this not ony to protect one user, but to protect all users by preventing VAC from sigging the dll.

so in short, you may use modified version of osiris (just any amount of modification, even if one line of working code) this will work with you for a few months, but that won't prevent VAC from sigging the dll and wait for you to got ow ban then vac ban.

use vac bypass and inject any dll with any injector detected or not, you won't get vac banned forever.

**Comment by popoyeLetal** [comment]
Thanks for the long answer!!! Got everything clear! 
I've been always using the VAC Bypass but im always afraid just one day stops working.... So thats why just in case i use the private dll (not dll) and injector. Just for sure i not get VAC banned. About the ow I feel safe as i use it totaly legit, the hack radar, sound options, rescale 1.33 and some minor fixes. Osiris has a great base, but if you dont do nothing more than coding o even downloading it coded if VAC bypass stops working (or u forgot to activate it)....
Just in case i dont store localy any dll, always in flash mem, i rembered reading steam can see archives under C:/ drive.
Thanks!!!

**Comment by linuxer9** [comment]
your welcome, I personally use [this vac bypass](https://github.com/zyhp/vac3_inhibitor), as it disables all modules, I think that daniel's vac bypass doesn't disable all modules, but it's still safe to use for now.

the only thing you should worry about for now, is overwatch, as it has returned aggressively. I recommend using low fov aimbot and triggerbot with a delay, no wallhacks or radar.

**Comment by popoyeLetal** [comment]
Thanks!! I will change the vacbypass for sure,been with daniels for some months without problem, but that looks much better!
Any problems with radar??

**Comment by linuxer9** [comment]
you're welcome, what I meant to say don't be obvious when you are using them
